data/entities.json5

```diff
@@ -11422,5 +11422,14 @@
         "api.usersnap.com",
         "cdn.usersnap.com",
     ]
+  },
+  {
+    "name": "Rewardful",
+    "homepage": "https://www.getrewardful.com",
+    "categories": ["analytics"],
+    "domains": ["r.wdfl.co"],
```

Comment:
do they own the whole domain such that `*.wdfl.co` is appropriate?

Reply:
`*.wdfl.co` will work, I thought it was a bit over the top but all the others are done that way so 👍